<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MQTTWillSettingsWidget</class>
 <widget class="QWidget" name="MQTTWillSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QLabel" name="lWillQoS">
     <property name="text">
      <string>QoS</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QListWidget" name="lwWillStatistics">
     <item>
      <property name="text">
       <string>Minimum</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Maximum</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Arithmetic mean</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Geometric mean</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Harmonic mean</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Contraharmonic mean</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Median</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Variance</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Standard deviation</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mean deviation</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mean deviation around median</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Median deviation</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Skewness</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Kurtosis</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Entropy</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="cbWillQoS">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lWillOwnMessage">
     <property name="text">
      <string>Own message:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lWillMessageType">
     <property name="text">
      <string>Message type:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="cbWillUpdate">
     <item>
      <property name="text">
       <string>Time Interval</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>On Click</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="leWillOwnMessage"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lWillUpdateInterval">
     <property name="text">
      <string>Time interval:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="leWillUpdateInterval">
     <property name="text">
      <string>10000</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbWillMessageType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Own Message</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Statistics</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Last Message Received</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="chbWillRetain">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lUseRetainMessage">
     <property name="text">
      <string>Retain:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lWillStatistics">
     <property name="text">
      <string>Will statistics:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lWillUpdateType">
     <property name="text">
      <string>Update type:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QPushButton" name="bApply">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cbWillTopic">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lWillTopic">
     <property name="text">
      <string>Topic:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lEnabled">
     <property name="text">
      <string>Enabled:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="chbEnabled">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
